src/main/java/com/hackclub/hccore/commands/StatsCommand.java

```diff
@@ -77,5 +148,41 @@ private void sendStatistics(CommandSender sender, Player player) {
         SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
         sender.sendMessage(
                 "- Registered since: " + dateFormat.format(new Date(player.getFirstPlayed())));
+
+        if (extended) {
```

Comment:
I feel like the extended option should have more stats in it. Otherwise, this could just be merged into the regular message.